components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -1232,6 +1234,11 @@ constexpr char kSolanaBuyTokenFeeBps[] = "85";
 constexpr char kSolanaFeeRecipient[] =
     "3NUW8hWoCnLgJwWCVnwdFo2Dsz8bKwLac9A3VgS2jLUQ";
 
+// Blowfish simulations constants
+constexpr char kBlowfishBaseAPIURL[] = "https://blowfish.wallet.brave.com";
```

Comment:
This adds networking and AFAIK needs formal S&P review.

Reply:
https://github.com/brave/security/issues/1105 is already done, pending decision on opt-in vs opt-out and privacy policy wording (not blocking this PR). I'm adding Sec team as reviewer for a formal code-review.